src/main/g8/build.sbt

```diff
@@ -24,6 +45,12 @@ scalacOptions ++= {
   }
 }
 
+scalacOptions in Test += "-Yrangepos"
+
+scalacOptions in (Compile, console) ~= (_ filterNot (Set("-Xfatal-warnings", "-Ywarn-unused-import").contains))
+
+scalacOptions in (Test, console) := (scalacOptions in (Compile, console)).value
```

Comment:
Won't this override line 48?  I have to admit to not actually knowing how SBT linearizes these sorts of declarations.